doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -160,15 +173,26 @@ to run your project both in the traditional way and in the Flower Next way:
   |startserver_link|_. Here's an example:
 
 .. code-block:: python
-    :emphasize-lines: 2,9
+    :emphasize-lines: 2,5,11
```

Comment:
```suggestion
    :emphasize-lines: 2,8
```

Reply:
Yes, it seems more reasonable to highlight both the `client_fn` and `server_fn` and how each respective `Client` and `Server` object is constructed.